### ISSUE TITLE
PanelBar: Elements with role "menuitem" are not expected to have "aria-expanded" attribute

### ISSUE THREAD

**Issue by veselints** [telerik/kendo-ui-core#5954]
### Bug report

Currently, the items in the PanelBar have `menuitem` role. They also have `aria-expanded` attribute. As per the WAI-ARIA specification (https://www.w3.org/WAI/PF/aria/roles#menuitem), elements with role `menuitem` are not expected to have `aria-expanded` attribute.

related to: #5955 

### Reproduction of the problem
Review the generated HTML on the PanelBar Index demo:
https://demos.telerik.com/kendo-ui/panelbar/index

### Expected/desired behavior
We should reconsider the use of the `menuitem` role or the use of the `aria-expanded` attribute.

### Environment
* **Kendo UI version:** 2020.2.617
* **Browser:** [all]

**Comment by veselints** [comment]
Also reported in: Ticket ID: 1481206

**Comment by veselints** [comment]
The current and also the following bug (#5955 ) are results of the fact that the PanelBar rendering (and aria attributes) do not follow the recommendation of the WAI-ARIA Practices(https://www.w3.org/TR/wai-aria-practices-1.1/#accordion). Following those, the Accordion (PanelBar) should:
* The title of each accordion header is contained in an element with role `button`.
* Each accordion header button is wrapped in an element with role `heading` that has a value set for aria-level that is appropriate for the information architecture of the page.
* The button element is the only element inside the heading element. That is, if there are other visually persistent elements, they are not included inside the heading element.
* If the accordion panel associated with an accordion header is visible, the header button element has `aria-expanded` set to `true`. If the panel is not visible, aria-expanded is set to false.
* The accordion header button element has `aria-controls` set to the ID of the element containing the accordion panel content.

With the current implementation of the Kendo PanelBar, we do not cover any of the above recommendations of the WAI-ARIA practices. Apart from that:
* We use role `menu` on the PanelBar. Such a role is not appropriate because apart from the navigation links, the widget also hosts content. According to the WAi-ARIA practices, no role should be used on the component;
* As a consequence of the above, we use role `menuitem` on the action links that open/collapse content elements. That, however, contradicts the mandatory for the Accordion-like components `aria-expanded` attribute used on the same elements.
* We manage the tab navigation of the links/buttons of the PanelBar. That is due to the fact that the whole widget gets focused. This, however, contradicts the concept of leaving all button elements in the tab sequence of the page, which is advocated in the WAI-ARIA practices example (https://www.w3.org/TR/wai-aria-practices-1.1/examples/accordion/accordion.html).

**Comment by veselints** [comment]
`menu` and `menuitem` roles have been changed to `tree` and `treeitem` respectively. The `region` role has been removed. Here are the screen readers changes after that change:

### NVDA in Firefox
* The role of the widget is announced as a `treeview`;
* The level of the currently focused item is announced;
* The nested items within an expanded item are not announced when their parent gets focus;

### ChormeVox in Chrome
No changes

### JAWS in IE 11
* The level is announced when navigating from one level to another - when the first subitem is focused, after pressing the up arrow and focusing the parent, its level gets announced. The same happens when going the other way;
* The state (open or closed) is announced when navigating to an item with content/subitems;
